physics/column_model_mod.F90

```diff
@@ -69,10 +69,10 @@ subroutine InitColumnModel(elem, cm,hvcoord,hybrid,tl,nets,nete,runtype)
                    elem(ie)%state%Qdp(i,j,k,1,1:2)      = 0_real_kind
 
 #ifndef USE_KOKKOS_KERNELS
-                   elem(ie)%derived%FQ(i,j,k,1,1:3)     = 0_real_kind
+                   elem(ie)%derived%FQ(i,j,k,1)     = 0_real_kind
```

Comment:
i probably missed a previous PR. where was all this stuff in physics with pragmas USE_KOKKOS introduced?

Reply:
never mind -- i am just not familiar with physics folder.